src/components/Card/Card.js

```diff
@@ -10,23 +22,68 @@ const baseStyles = ({ theme }) => css`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  ${shadowSingle({ theme })};
-  min-width: 460px;
-  padding: ${theme.spacings.mega} ${theme.spacings.giga};
+  min-width: 400px;
 `;
 
+const shadowStyles = ({ theme, shadow }) => {
+  const shadows = {
+    [SINGLE]: shadowSingle({ theme }),
```

Comment:
Maybe instead of executing the functions, just leave them here without executing. Then, down in the `css` call you only execute once.

```javascript
const shadowFns = {
  [SINGLE]: shadowSingle,
  [DOUBLE]: shadowDouble,
  [TRIPLE]: shadowTriple,
};

return css`
  ${shadowFns[shadow]({theme})}
`;
```

Reply:
Very good point.